src/Core/Application/IO/LoggingManager.hpp

```
@@ -160,10 +162,53 @@ class Log {
+
+		// Get message type
+		std::string displayType = "Unknown message type";
+		LogColor(type, displayType);
+
+		// Get thread info
+		std::string threadInfo = "";
+		LogThreadInfo(threadInfo);
+
+		// Log
+		std::string fmt = std::vformat(messageFmt, std::make_format_args(args...));
+		std::string logMsg = std::format(
```

Clang (14.0.6)

[warning] variable 'logMsg' is not initialized

(cppcoreguidelines-init-variables)